Directory.Build.props

```diff
@@ -71,6 +74,9 @@
     <DotNetRoot Condition="!HasTrailingSlash('$(DotNetRoot)')">$(DotNetRoot)\</DotNetRoot>
     <DotnetCliPath Condition="'$(DotnetCliPath)'==''">$(DotNetRoot)</DotnetCliPath>
     <ToolHostCmd Condition="'$(ToolHostCmd)'==''">"$(DotNetRoot)dotnet"</ToolHostCmd>
+
+    <!-- Need to keep in sync with Tools.props file. -->
+    <AnalyzerPropsFile>$(ToolSetCommonDirectory)Tools.Analyzers.props</AnalyzerPropsFile>
```

Comment:
Is there any way we can share this property with Tools.props?  Otherwise, if you need to keep in sync, please create it as a property in the other file.